app/services/delete_time.rb

```diff
@@ -0,0 +1,9 @@
+class DeleteTime
+  def initialize(time_entries)
+    @time_entries = time_entries
+  end
+
+  def delete_total_time
+    @time_entries.delete_if { |hash| hash['name'] == 'Total time' }
```

Comment:
I would use `reject!` instead of `delete_if` because it better indicates that the callee is being changed.